etc/docker/kernel-tf-py/Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV KERNEL_LANGUAGE python
 ADD jupyter_enterprise_gateway_kernel_image_files*.tar.gz /usr/local/bin/
 
 RUN conda install --quiet --yes \
+    ipykernel \
```

Comment:
Since this is an image from the [jupyter-stacks repo](https://github.com/jupyter/docker-stacks), I think it would be best to just bump the tag on the `FROM` image to include an updated image that would have `ipykernel` >= 6.0.  Any of their more recent images should suffice.